### ISSUE TITLE
CI is failing

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaCI/Coverage.jl#283]
CI is failing on #281 and #282 for reasons that seem unrelated to the pull requests.

@vtjnash Any idea what is going on?

**Comment by vtjnash** [comment]
Duplicate of https://github.com/JuliaCI/Coverage.jl/pull/280 ?

**Comment by DilumAluthge** [comment]
Hmmm I'm not sure. Tests in #280 are also failing, I think with the same error.

**Comment by fingolfin** [comment]
In Julia 1.5, tests are failing because coverage for `test/data/testparser.jl` changed:
1. its second line is now counted as being executed once, not twice (which seems correct to me? I.e. the prior behavior was wrong?)
2. the final (uncovered, empty) line at the end is no longer in the coverage data (again, seems like a bug fix to me)

To deal with those two, once could modify `target = Coverage.CovCount[nothing, 2, nothing ...` by adding a Julia version based test, and for version 1.5 or later, change the 2 to 1 and remove the last entry of the array.

Anyway, that's the easy one. The other issue which affects all Julia versions, seems to be about the coverage reporting for this package itself: 
```
ERROR: LoadError: HTTP.ExceptionRequest.StatusError(422, "POST", "/api/v1/jobs", HTTP.Messages.Response:
"""























{"message":"service_job_id (429306341) must be unique for Travis Jobs not supplying a Coveralls Repo Token","error":true}""")
Stacktrace:
```

So I guess somebody needs to look into fixing the Coveralls support?

**Comment by fingolfin** [comment]
Fixed by PR #289
